### PR TITLE
access registers with PT_REGS macros & add LINUX_ARCH flags

### PR DESCRIPTION
**Pull request by chriskaliX**
access registers with PT_REGS macros and add LINUX_ARCH flags to makefile.

### CODE REVIEW

File: kern/mysqld56_kern.c [hunk 1]
```diff
@@ -44,7 +44,7 @@ int mysql56_query(struct pt_regs *ctx) {
         return 0;
     }
 
-    uint64_t len  = (uint64_t) (ctx)->cx;
+    uint64_t len  = (uint64_t)PT_REGS_PARM1(ctx);
```
Comment (cfc4n) [inline kern/mysqld56_kern.c:47]: (ctx)->cx macros is PT_REGS_PARM4 .

Comment (chriskaliX) [reply kern/mysqld56_kern.c:47]: Yes! it's a mistake 👍

**Comment by chriskaliX** [comment]
it's updated now

**Comment by cfc4n** [comment]
LGTM, thanks.

**Pull request merged**